### PR TITLE
complete source code for code coverage project for Knative repos

### PR DESCRIPTION
**Pull request by steuhs**
Fixes #

### CODE REVIEW

**Comment by steuhs** [comment]
> /hold
> 
> Please put it inside tools/, not at the root level.

@adrcunha done

File: coverage/main_test.go [hunk 1]
```diff
@@ -0,0 +1 @@
+package main
```
Comment (adrcunha) [inline coverage/main_test.go]: This seems to be a leftover.

**Comment by adrcunha** [comment]
Note: as previously agreed, the code in this PR was not reviewed. That was a one-time exception so we could get the source out of Stephen's personal repository and into test-infra ASAP, as properly reviewing it would take some time.

**Pull request merged**